public/index.html

```diff
@@ -23,6 +23,12 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.0/css/materialize.min.css" rel="stylesheet" />
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" />
+
+      <style>
+          .modal .modal-footer {
```

Comment:
could you move this to a separate css file

Reply:
It is resolved